udaykalra/walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -172,28 +199,20 @@ private int classifyOverlap(TimeRange overlappingTimeRange, TimeRange timeRange)
     if ((timeRangeStartMinute == overlappingStartMinute)
         && (timeRangeEndMinute > overlappingEndMinute)) {
       return OVERLAP_WITH_SAME_START;
-    }
-
-    else if ((timeRangeEndMinute == overlappingEndMinute)
+    } else if ((timeRangeEndMinute == overlappingEndMinute)
```

Comment:
Also here, don't need the parenthesis around the comparison expressions. See:

https://introcs.cs.princeton.edu/java/11precedence/